lib/ansible/plugins/filter/vault.yml

```diff
@@ -29,18 +29,41 @@ DOCUMENTATION:
         - This toggle can force the return of an C(AnsibleVaultEncryptedUnicode) string object, when V(False), you get a simple string.
         - Mostly useful when combining with the C(to_yaml) filter to output the 'inline vault' format.
       type: bool
-      default: False
+      default: false
 
 EXAMPLES: |
-  # simply encrypt my key in a vault
+  # Example 1: Encrypt a variable using the vault filter
   vars:
-    myvaultedkey: "{{ keyrawdata|vault(passphrase) }} "
+    keyrawdata: "my_secret_key"
+    passphrase: "my_vault_password"
+    myvaultedkey: "{{ keyrawdata | vault(passphrase) }}"
```

Comment:
or just put the strings directly in the filter, the example makes little sense with variable colocation

Reply:
done!